analytics/analytics-ui/src/app/results/table/table.component.html

```diff
@@ -0,0 +1,35 @@
+<!--Column Remove Selector and Button-->
+<div>
+  <mat-form-field style="margin-left:110px">
+    <mat-label>Columns</mat-label>
+    <mat-select [(value)]="selected">
+      <mat-option *ngFor="let column of columnsToDisplay" value={{column}}>
+        {{ column | uppercase }}
```

Comment:
Is there a need to uppercase the values? Generally people find it harder to scan uppercase text.